src/main/java/com/avairebot/blacklist/Blacklist.java

```diff
@@ -79,7 +79,7 @@ public void remove(long id) {
                 .where("id", id)
                 .delete();
         } catch (SQLException e) {
-            AvaIre.getLogger().error("Failed to sync blacklist with the database: " + e.getMessage(), e);
+            AvaIre.getLogger().error(String.format("Failed to sync blacklist with the database: %s", e.getMessage()), e);
```

Comment:
Why use string.format there, when the error message is appended to the end of the log message anyway?